test/yulPhaser/Mutations.cpp

```diff
@@ -434,6 +436,181 @@ BOOST_AUTO_TEST_CASE(fixedPointCrossover_should_always_use_position_zero_as_spli
 	BOOST_CHECK(crossover10(empty, splittable) == splittable);
 }
 
+BOOST_AUTO_TEST_CASE(randomTwoPointCrossover_should_swap_chromosome_parts_between_two_random_points)
```

Comment:
Since I don't really know the semantics of the crossovers I won't try to review this file

Reply:
Sure. Just in case though: I documented all the operators in [Mutations.h](https://github.com/imapp-pl/solidity/blob/yul-phaser-crossover-operators/tools/yulPhaser/Mutations.h). I'm not sure if it's explained well, but if not, I can try to improve it.